dev-java/joda-time/joda-time-2.10.10.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+# Skeleton command:
+# java-ebuilder --generate-ebuild --workdir . --pom joda-time-2.10.10/pom.xml --download-uri https://github.com/JodaOrg/joda-time/archive/refs/tags/v2.10.10.tar.gz --slot 0 --keywords "~amd64 ~ppc64 ~x86" --ebuild joda-time-2.10.10.ebuild
+
+EAPI=7
+
+JAVA_PKG_IUSE="doc source test"
+MAVEN_ID="joda-time:joda-time:2.10.10"
+JAVA_TESTING_FRAMEWORKS="junit"
+
+inherit java-pkg-2 java-pkg-simple
+
+DESCRIPTION="Date and time library to replace JDK date handling"
+HOMEPAGE="https://www.joda.org/joda-time/"
+SRC_URI="https://github.com/JodaOrg/${PN}/archive/refs/tags/v${PV}.tar.gz -> ${P}-sources.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+# Common dependencies
+# POM: ${P}/pom.xml
+# org.joda:joda-convert:1.9.2 -> >=dev-java/joda-convert-2.2.1:0
+
+CDEPEND="
```

Comment:
Change all CDEPEND with COMMON_DEPEND

Reply:
You want me to rework even accepted PRs like https://github.com/gentoo/gentoo/pull/20141?

Reply:
If so, that fix should go into `java-ebuilder`. You might want to file a bug there